src/include/reconcile.h

```diff
@@ -225,6 +225,11 @@ struct __wt_reconcile {
      * violation and fragile, we need a better solution.
      */
     WT_CURSOR_BTREE update_modify_cbt;
+
+    /* Variables to track pages reconciled with time window values and prepared transactions */
+    bool page_with_ts;
+    bool page_with_txn_id;
+    bool page_with_prepared_txn;
```

Comment:
I'm not really a fan of these variable names as we're really tracking that the cells were written with these values not that the page has those values, it could definitely be confusing given that the ref->addr has timestamps, txnids. What about `page_cell_rec_with_ts`. `page_cell_rec_with_txnid` etc?

Reply:
Fixed

Reply:
Perhaps in order to make the variable name somewhat consistent with the statistic name we should make the values `rec_page_cell_with_ts`, and then update the statistic to be the same for all of these values. @sulabhM what do you think?

Reply:
Maybe we can create an anonymous struct and put these variables in that struct, could be handy if we decide to pass __cell_pack_value_validity() an argument to be set.

Agreed can do with better naming, since we write these txnids/ts in page address as well. I am not good at suggesting names, how about "values_written_with_ts"?

Reply:
I will be okay with rec_page_cell_with_ts.

Reply:
I have made the names consistent with stats.